### ISSUE TITLE
Question on BufferSlice.next

### ISSUE THREAD

**Issue by dodysw3** [cloudwego/shmipc-spec#6]
https://github.com/cloudwego/shmipc-spec/blob/51faf9baa7dd10653180ed1aff5746567792424a/README.md?plain=1#L189
> A pointer to the next BufferSlice in the linked list, as an offset from the start of the BufferManager in shared memory.

However the provided example shown below indicates that it is an offset from the start of BufferList?

**Comment by zhquzzuli** [comment]
yes, example shown below only shows the structure of a BufferSlice，the actual offset value in code will plus BufferList header length and BufferManager header length.
